src/cata_tiles.cpp

```diff
@@ -3799,6 +3840,37 @@ bool cata_tiles::draw_zombie_revival_indicators( const tripoint &pos, const lit_
     return false;
 }
 
+void cata_tiles::draw_zlevel_overlay( const tripoint &p, const lit_level ll,
+                                      color_block_overlay_container &color_blocks )
+{
+    if( is_isometric() ) {
+        return;
+    }
+    // Draw tileset fog sprite if exists
+    if( tileset_ptr->find_tile_type( "zlevel_fog" ) ) {
+        draw_from_id_string( "zlevel_fog", TILE_CATEGORY::NONE, empty_string, p, 0, 0, ll, false );
```

Comment:
Does this support tile variants? A single fog sprite might look very repetitive.

Reply:
It does now.  Thanks for pointing it out.